drivers/at86rf2xx/include/at86rf2xx_internal.h

```diff
@@ -100,9 +120,16 @@ void at86rf2xx_reg_write(const at86rf2xx_t *dev, uint8_t addr, uint8_t value);
  * @param[out] data     buffer to read data into
  * @param[in]  len      number of bytes to read from SRAM
  */
+#if defined(MODULE_AT86RFA1) || defined(MODULE_AT86RFR2)
+static inline void at86rf2xx_sram_read(const at86rf2xx_t *dev, uint8_t offset,
+                         uint8_t *data, size_t len) {
```

Comment:
Please align the parameters of the function (add more spaces before `uint8_t *data`)